simpleimage/make_rootfs.sh

```diff
@@ -49,12 +60,23 @@ fi
 
 TEMP=$(mktemp -d)
 cleanup() {
-	if [ -e "$DEST/proc/cmdline" ]; then
-		umount "$DEST/proc"
-	fi
-	if [ -d "$DEST/sys/kernel" ]; then
-		umount "$DEST/sys"
-	fi
+	set +e
+	while [ -e "$DEST/proc/cmdline" ]; do
+		umount $DEST/proc
+		sleep 2
+	done
+	while [ -d "$DEST/sys/kernel" ]; do
+		umount $DEST/sys
+		sleep 2
+	done
+	while [ -e "$DEST/dev/pts/0" ]; do
+		umount $DEST/dev/pts
+		sleep 2
+	done
+	while [ -e "$DEST/dev/cpu" ]; do
+		umount $DEST/dev
+		sleep 2
+	done
```

Comment:
These changes potentiall never return. If for whatever reason umount fails the script should just continue and not wait forever.